packages/blockchain-wallet-v4/src/network/api/http.ts

```diff
@@ -0,0 +1,100 @@
+import { merge, mergeRight, path, pathOr, prop } from 'ramda'
+import axios, { AxiosRequestConfig } from 'axios'
+import queryString from 'query-string'
+
+axios.defaults.withCredentials = false
+axios.defaults.timeout = Infinity
+
+interface RequestConfig extends AxiosRequestConfig {
+  contentType?: string
+  endPoint?: string
+  sessionToken?: string
+  url?: string
+}
+
+interface GetConfig extends RequestConfig {
+  ignoreQueryParams: boolean
+}
+
+type Header = {
+  Authorization?: string
+  ['Content-Type']: string
+}
+
+export default ({ apiKey }: { apiKey: string }) => {
+  const encodeData = (data: any, contentType: string) => {
```

Comment:
I've set `data` to `any` as that is how it's defined in `AxiosRequestConfig`.
I'm unsure what would be sent through but it could possibly be updated to something like `[k:string]: string | number | object`

Reply:
I think `any` is fine for now. I cant recall off the top of my head if we even use this field beyond the defaults. @plondon do you?